server/container/guice/testing/pom.xml

```diff
@@ -34,5 +34,6 @@
 
     <modules>
         <module>custom-mailets</module>
+        <module>dependency</module>
```

Comment:
do you really have to add it?

Reply:
To ensure that we don't break compilation randomly?